app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -108,18 +111,23 @@ public void onPageFinished(WebView view, String url) {
                     public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                         FragmentTransaction transaction =
                                 getSupportFragmentManager().beginTransaction();
+                        Bundle bundle = new Bundle();
```

Comment:
I've been doing more checking on the actual Analytics on the Firebase page and it turns out this time parameter that I put into the bundle doesn't provide any use (Firebase already keeps track of time). Let's just remove all of these bundle declarations and just do mFirebaseAnalytics.logEvent(eventName, null) for now